qa/smoke-test-http/build.gradle

```diff
@@ -34,4 +34,9 @@ integTest {
    * other if we allow them to set the number of available processors as it's set-once in Netty.
    */
   systemProperty 'es.set.netty.runtime.available.processors', 'false'
+
+  /* We need to set this here to prevent release builds from failing while the default is different between snapshot and release builds.
+  * Specifically, this is needed for SystemIndexRestIT, which assumes the value is `deprecated`.
+  */
+  systemProperty 'es.system_index_access_behavior', 'deprecated'
```

Comment:
I am not sure about this; shouldn't this be an `assume` in the tests?

Reply:
I've removed the system property in these gradle files and used assumeTrue in the tests in question.